src/Bot.hs

```diff
@@ -52,11 +62,23 @@ humanFriendlyUrl x = unpack $ mconcat [baseUrl, "?hoogle=", x,"&start=1&count=1"
 baseUrl :: Text
 baseUrl = "https://www.haskell.org/hoogle/"
 
+hayooUrl :: Text -> String
+hayooUrl x = unpack $ mconcat ["http://hayoo.fh-wedel.de/json?query=" , x]
```

Comment:
Now that you've got `st` in your dependencies, I like to use it for these cases too

```hs
hayooUrl x = [st|http://...#{x}|]
```